requirements.txt

```diff
@@ -38,3 +38,4 @@ sentry-sdk==2.17.0
 slackclient==2.9.4
 urllib3==2.2.3
 uvicorn[standard]==0.28.0
+setuptools==70.0.0
```

Comment:
@Emterry where did this come from? I'm not sure that this should be a top level dependency

Reply:
morning @michaeljcollinsuk sorry missed this yesterday - it showed up as a critical in one of the trivy scans I couldnt identify what dependancy it was coming from so added it in the requirements. I can remove and put a trivy ignore instead?

Reply:
Thats interesting, is there a way I can see the trivy warning?

Reply:
If we do need to pin it, can we use the latest version? https://pypi.org/project/setuptools/